vortex-array/src/array/chunked/canonical.rs

```diff
@@ -34,10 +34,6 @@ pub(crate) fn try_canonicalize_chunks(
     validity: Validity,
     dtype: &DType,
 ) -> VortexResult<Canonical> {
-    if chunks.is_empty() {
```

Comment:
this was a leftover from a time before `dtype` was able to be passed into this method, we had needed to examine the first chunk to get the DType but that's no longer the case